api/cost/consume-cost-response.ts

```diff
@@ -0,0 +1,18 @@
+import { CostResponse } from "../../models/cost-jobs";
+import { QueueConsumer } from "../util/queueConsumer";
+
+const COST_RESPONSE_QUEUE_URL = process.env.COST_RESPONSE_QUEUE_URL ?? "";
+
+class ConsumeCostResponse extends QueueConsumer<CostResponse> {
+  processMessage(message: string | undefined): CostResponse {
+    const { code, content } = JSON.parse(message ?? "");
```

Comment:
When `message` is undefined, `JSON.parse("")` will result in a syntax error. Maybe
```suggestion
    const { code, content } = message ? JSON.parse(message): {};
```

But that will end up returning `{ code: undefined, content: undefined }` which doesn't really feel correct either (because that's basically just `{}` which doesn't meet the interface).